products/sle/main.pm

```diff
@@ -787,23 +781,25 @@ elsif (get_var('XFSTESTS')) {
         loadtest 'kernel/change_kernel';
     }
     prepare_target;
-    if (get_var('XFSTESTS_SKIP_INSTALL', 0) || check_var('XFSTESTS', 'installation') || is_pvm || check_var('ARCH', 's390x')) {
+    if (get_var('XFSTESTS_INSTALL', 0) || check_var('XFSTESTS', 'installation') || is_pvm || check_var('ARCH', 's390x')) {
         loadtest 'xfstests/install';
-        unless (check_var('NO_KDUMP', '1')) {
-            loadtest 'xfstests/enable_kdump';
-        }
-        if (get_var('XFSTEST_KLP')) {
-            loadtest 'kernel/install_klp_product';
-        }
-        if (check_var('XFSTESTS', 'installation')) {
-            loadtest 'shutdown/shutdown';
-        }
-        else {
-            load_xfstests_tests();
-        }
     }
     else {
-        load_xfstests_tests();
+        set_var('NO_KDUMP', '1') if !get_var('NO_KDUMP');
+    }
+    unless (check_var('NO_KDUMP', '1')) {
```

Comment:
This one needs to check when KDUMP is enabled in both install and subtests, it may have an issue. Although we could only set NO_KDUMP in create_hdd_xfstests job and XFSTESTS_INSTALL=1 job, but maybe here could check an "and" condition check_var('NO_KDUMP', '1') && (check_var('XFSTESTS', 'installation') || get_var('XFSTESTS_INSTALL')) to avoid mistake setting.

Reply:
For installation, the logic unchanged.
For subtests, we must set NO_KDUMP=1 if no this parameter, or else enable kdump by default.